app/src/androidTest/java/com/processout/processoutexample/ExampleInstrumentedTest.java

```diff
@@ -1,12 +1,14 @@
 package com.processout.processoutexample;
 
 import android.content.Context;
-import android.support.test.InstrumentationRegistry;
-import android.support.test.runner.AndroidJUnit4;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import androidx.test.InstrumentationRegistry;
```

Comment:
@adamszewe According to the Android API Reference, [`androidx.test.InstrumentationRegistry`](https://developer.android.com/reference/androidx/test/InstrumentationRegistry) is deprecated in favour of [`androidx.test.platform.app.InstrumentationRegistry`](https://developer.android.com/reference/androidx/test/platform/app/InstrumentationRegistry). Is there a specific reason you used `androidx.test.InstrumentationRegistry` here?

Reply:
Nope, probably I took the first option that IDE suggested, this is a dummy example test anyway, but I fixed that as well.